cassiopeia/simulator/Cas9LineageTracingDataSimulator.py

```diff
@@ -93,6 +93,9 @@ class Cas9LineageTracingDataSimulator(LineageTracingDataSimulator):
             Note that the numpy random seed gets set during every call to
             `overlay_data`, thereby producing deterministic simulations every
             time this function is called.
+        collapse_sites_on_cassette: Whether or not to collapse cuts that occur
```

Comment:
Can you add a brief sentence describing this function to the docstring? This can go at the end of the paragraph starting with "Second, the class accepts..."